doc/newfeat.texi

```diff
@@ -60,6 +60,12 @@ number of characters as their non-pretty cousins.
 @item Variable @code{ess-s-versions-list} is obsolete and ignored.
 Use @code{ess-s-versions} instead.  You may pass arguments by starting the inferior process with the universal argument.
 
+@item All of the @code{*-program-name} variables have been renamed to @code{*-program}.
```

Comment:
Well, this will mean a lot of users actually. I think we should alias. I am not sure about -program postfix either. I think `-executable` is way more common.  Let's see what others think.

Reply:
> I think we should alias

Sure, that's easy enough. My thinking in not doing so was to make them aware of it in the least painful way possible. I'll add the (obsolete?) aliases shortly though.

> I think -executable is way more common. 

On my system, I have more -program than -executable variables. I only have 8 `-executable`, and perhaps 50ish `-program` (not counting ESS's, of course).  Another reason I (slightly) prefer `-program` is because elisp automatically marks these as risky, whereas it doesn't with -executable.

At the end of the day though, I really don't care that much about the name.

Reply:
> Another reason I (slightly) prefer -program is because elisp automatically marks these as risky, whereas it doesn't with -executable.

I like this so I'd say to go with `-program`.